tests/Microsoft.Identity.Web.Test/AuthorityHelpersTests.cs

```diff
@@ -118,5 +118,623 @@ public void EnsureAuthorityIsV2(string initialAuthority, string expectedAuthorit
             options.Authority = AuthorityHelpers.EnsureAuthorityIsV2(options.Authority);
             Assert.Equal(expectedAuthority, options.Authority);
         }
+
+        [Theory]
```

Comment:
we don't use theorydata much in Id Web, just because it's hard to debug, it's more of SAL and wilson thing...but we'll live with it. :)

Reply:
😅